docker/Dockerfile.builder

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:20.04
```

Comment:
why not golang:xxx-bookworm?

Reply:
No reason actually seemed right then :) wanted to be more clear with dependencies
Now thinking that using golang image is better because is easier to change to ccx-go-builder if we need

I will change it, thanks